com.projectrk.launcher.yml

```diff
@@ -0,0 +1,232 @@
+id: com.projectrk.launcher
```

Comment:
I can't reach the website at projectrk.com

Reply:
It's project-rk.com

Reply:
I applied the fixes with latest commits